vendor/omnipay/sagepay/src/Message/AbstractRequest.php

```diff
@@ -352,6 +352,12 @@ public function sendData($data)
             case 'DeliveryCountry':
               $data[$key] = $data['BillingCountry'];
               break;
+            case 'NotificationURL':
```

Comment:
@mattwire I just added this to the extension - https://github.com/eileenmcnaughton/nz.co.fuzion.omnipaymultiprocessor/commit/763d01f992ed4004af1a406a69e5d0406ed459e0 - which makes this change redundant.

(also fyi @nganivet

Reply:
@eileenmcnaughton Ok, I agree it looks like `case 'NotificationURL'` is redundant but I need to pass in qfKey as an URL parameter to generate the redirect URLs correctly.  Currently I'm doing that directly in `CRM_Core_PaymentExtended::getNotifyURL`.